scripts/webpack.config.js

```diff
@@ -91,7 +91,7 @@ function getConfig(isReactExternalized) {
                     // For webpack dev build perf we want to exlcude node_modules unless we want to support legacy browsers like IE11
                     exclude: shouldIncludeAllSupportedBrowsers
                         ? /@babel(?:\/|\\{1,2})runtime|pikaday|core-js/
-                        : /node_modules/,
+                        : /node_modules\/(?!@box\/cldr-data)/, // Exclude node_modules except for @box/cldr-data which is needed for styleguidist
```

Comment:
What about this particular change necessitated this new configuration?

Reply:
No other code in the repo had actually called [numAbbr](https://github.com/box/box-ui-elements/blob/master/src/utils/numAbbr.ts)

Reply:
It increases the bundle size for the sidebar.js from `1.57MB` to `1.61.MB` which seems ok. It should work with both versions of react-intl because it shortcuts both and uses `IntlMessageFormat` to abbreviate the number and when we fully support react-intl v3 we can actually migrate over to using `<FormattedNumber />` with the `compact` notation